packages/wk-lib-consumer-playground-npm/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@bnayae/wk-lib-playground": "^1.0.45",
+    "@weknow-network/wk-lib-playground": "^1.0.45",
```

Comment:
All references in the project are scored to this @weknow-network so I updated them from bnayae

This is controlled by the "name" field in the package.json files if you ever need to change it